umi_tools/whitelist_methods.py

```diff
@@ -100,6 +100,9 @@ def getKneeEstimateDensity(cell_barcode_counts,
 
     if plotfile_prefix:
 
+        # require to run on systems with no X11
```

Comment:
@sshen8 How would you feel about moving lines 11 and 12 to just here (line 102). This way, if the user changes any params after importing umi_tools, then those changes will get captured.